tests/unit/literaturesuggest/test_literaturesuggest_tasks.py

```diff
@@ -94,6 +94,53 @@ def test_new_ticket_context():
     assert expected == result
 
 
+def test_new_ticket_context_handles_unicode():
+    data = {
+        'titles': [
+            {
+                u'title': 'Actual title contained: é',
```

Comment:
You have the actual string that caused the error on Sentry, why invent another? Just use "Chocs caractéristiques et ondes simples exceptionnelles pour les systèmes conservatifs à intégrale d'énergie: forme explicite de la solution".